paper-ripple.html

```diff
@@ -476,6 +493,13 @@
         return false;
       },
 
+      simulatedRipple: function() {
+        this.mousedownAction();
+        this.async(function() {
+          this.mouseupAction();
+        }, 1);
```

Comment:
To be honest, I would rather we fix `this.async` so that the onus is not on the user to document its eccentric behavior every time it is used. See https://github.com/Polymer/core-focusable/pull/12#discussion_r26275349 for context.

@kevinpschaaf @sjmiles @sorvell Do any of you guys have thoughts on this?